### PR TITLE
[New] Added package json for React-is

### PR DESCRIPTION
**Pull request by NielsRavn**
# Pull request template for new library

Styled-components v5 which is in this cdnjs is dependent upon react-is which is not in cdnjs, which i why im trying to add the library.

Pull request for issue: #
Related issue(s): # remake of pull request #13697

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/facebook/react
* Official website (optional, not the repository): https://reactjs.org/
* NPM package url (optional): https://www.npmjs.com/package/react-is
* License and its reference: MIT
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 6.600
  * Count of stars: 144.000
  * Count of forks: 27.700
* NPM download stats (optional):
  * Downloads in the last day: 
  * Downloads in the last week: 10,189,877
  * Downloads in the last month:

## Essential checklist

* [ ] I'm the author of this library
  * [ ] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [ ] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/react-is/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "react-is",
+  "version": "16.12.0",
```
Comment (MattIPv4) [inline ajax/libs/react-is/package.json]: Please don't include the version attr. The auto-update bot will populate this once it pulls in versioned files.

File: ajax/libs/react-is/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/facebook/react.git",
+    "directory": "packages/react-is"
+  },
+  "npmName": "react-is",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
```
Comment (MattIPv4) [inline ajax/libs/react-is/package.json]: No `dist` directory appears to be present in the most recent package version:

```
node_modules/react-is
├── LICENSE
├── README.md
├── build-info.json
├── cjs
│   ├── react-is.development.js
│   └── react-is.production.min.js
├── index.js
├── package.json
└── umd
    ├── react-is.development.js
    └── react-is.production.min.js
```

File: ajax/libs/react-is/package.json (hunk 1 above)
Comment (NielsRavn) [reply ajax/libs/react-is/package.json]: I have removed the version

File: ajax/libs/react-is/package.json (hunk 2 above)
Comment (NielsRavn) [reply ajax/libs/react-is/package.json]: It was copied from the react library setup here in the project, the react package has the same structure when installed.
I have removed the dist file path.

**Pull request merged**